### PR TITLE
Update cpnp for go 1.23

### PR DESCRIPTION
**Pull request by markovichecha**
When using Go 1.23 capnproto throws an error. I bumped dependency so the issue is resolved.

https://github.com/capnproto/go-capnp/issues/575

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
-[![Tests Status](https://github.com/enviodev/hypersync-client-go/actions/workflows/test.yml/badge.svg)](https://github.com/enviodev/hypersync-client-go/actions/workflows/test.yml)
```
Comment (0x19) [inline README.md]: This is issue. Can you please ensure that enviodev namespace is not modified.

File: arrow/reader.go [hunk 2]
```diff
@@ -2,16 +2,17 @@ package arrowhs
...
+	"math/big"
+
+	"capnproto.org/go/capnp/v3"
 	"github.com/apache/arrow/go/v10/arrow"
 	"github.com/apache/arrow/go/v10/arrow/ipc"
-	hypersynccapnp "github.com/enviodev/hypersync-client-go/capnp"
-	"github.com/enviodev/hypersync-client-go/types"
 	"github.com/ethereum/go-ethereum/common"
+	hypersynccapnp "github.com/markovichecha/hypersync-client-go/capnp"
+	"github.com/markovichecha/hypersync-client-go/types"
```
Comment (0x19) [inline arrow/reader.go]: Same here. Namespace is being modified.

File: arrow/response.go [hunk 3]
```diff
@@ -1,8 +1,9 @@
 package arrowhs
 
 import (
-	"github.com/enviodev/hypersync-client-go/types"
 	"math/big"
+
+	"github.com/markovichecha/hypersync-client-go/types"
```
Comment (0x19) [inline arrow/response.go]: Same.

**Review by 0x19 — CHANGES_REQUESTED** [review]
In general great! Can you please make sure that enviodev namespace is not modified across the code. Otherwise I'd approve and merge the PR. Can you please fix that?

**Comment by markovichecha** [comment]
Oh, that was fast. Thanks, I did changes on main so fork will work on my side, didn't realize it would affect this PR. I'll fix that in a few hours.

**Comment by markovichecha** [comment]
@0x19 I reverted my commit, you can merge it now, thanks!

**Review by 0x19 — APPROVED** [review]
This now looks good! Thanks!

**Pull request merged**